.env.local

```
@@ -0,0 +1,2 @@
+FEAT_AREAS=1
```

I suggest enabling the feature for all organizations.

```suggestion
FEAT_AREAS=*
```

---

Agreed, changed as such and added to .env.development instead.